website/integrations/services/fortigate-ssl/index.md

```diff
@@ -0,0 +1,239 @@
+    -   Property mapping:
+        -   authentik default SAML Mapping: Username
+        -   authentik default SAML Mapping: Groups
+    -   Named Property Mapping: Empty (------)
+    -   Assertion valid not before = minutes=5
+    -   Assertion valid not on or after = minutes=5
+    -   Session valid not on or after = (Set how long you want the user's session to be valid)
+    -   Default relay state = empty
+    -   Digest algorithm = sha256
+    -   Signature algorithm = sha256
+
+## Application section
+
+Lets create the application and link it to the provider
+
+### Create user group
+
+This is the user group that you matched on in the FortiGate "firewall group" above.
+
+-   Go to Directory -> Groups
+-   Click Create
+-   Name = `ak.user.group`
+-   Open ak.user.group and add the users whom should have access to the sslvpn
+-   save the group
+
+### Create the application
+
+> [!NOTE]
+> The Launch URL = blank://blank will prevent authentik from displaying it on the user's login page in authentik.
+
+-   Go to Applications -> Applications
+-   Name = What ever you fancy e.g. FGT-SSLVPN
+-   Slug = app.slug.name
+-   Group = empty (------)
+-   Provider = The provider you created before e.g. "FGT SSL SAML Provider"
+-   Backchannel Provider = empty (-----)
+-   Policy engine mode = any
+-   Launch URL = blank://blank
+-   Open in new tab = disabled
+-   icon = None
+-   Publisher = None
+-   Description = None
+-   Save
+
+### Limiting the access based on authentik group
+
+-   Open the application again
+-   Click on "Policy / Group / User Binding"
+-   Click bind existing policy
+-   Click on the "TAB" Group at the top
+-   Click on Select an object next to Group
+-   Chose ak.user.group
+-   Make sure enabled is chosen
+-   Order = 10
+-   Timeout = 30
+-   Failure result = Don't pass
+-   Click create
+
+You should now be able to login by selecting SSO login either on the portal or in FortiClient, depending on your portal configuration.
+
+> [!NOTE]
+> If you are using FortiClient remember to set the sslvpn profile to use single-signon either creating a manual profile or edit the profile in your EMS.
+
+## Troubleshooting
+
+These are just suggestions of what **could** be the cause of an issue and how to enable debug on the FortiGate
+
+> [!CAUTION]
+> Debugging can generate heavy load on a FortiGate firewall, so make sure your firewall is not already struggling with performance before you enable debugging, and remember to disabled it again when you are done.
+>
+> You can disable the debug with these commands.
+> `diag debug disable` > `diag debug reset`
+
+### Enabling debug output
+
+Before you can see any output you need to enable the debug
+`diagnose debug enable`
+
+### Debug saml daemon
+
+This will provide all possible output from the saml daemon.\
+`diag debug application samld -1`
+
+### Debug sslvpn (optional)
+
+This will provide insight into what happens when you use FortiClient, usually combined with salmd debug.\
```

Comment:
```suggestion
This will provide insight into what happens when you use FortiClient, usually combined with `salmd debug`.
```

Reply:
I'm not actually sure if the backslashes are intentional? Is this a way to format spacing?

Reply:
It was to have the quoted command on the next line, so each debug command would be more visable, and the explanation would be a kind of heading for it.

Reply:
Haha :D DragonWin did it inline before we decided to make it in a seperate line. But at the end it doesn't matter, we thought it is better for a random user to read it as the command itself if it is on the next line.